### ISSUE TITLE
 Advanced connectivity policy tests script could not be downloaded

### ISSUE THREAD

**Issue by ThilinaManamgoda** [Azure/SQL-Connectivity-Checker#59]
When the script is invoked the following error is thrown at the end of the execution,
```
 Advanced connectivity policy tests script could not be downloaded!
 Confirm this machine can access https://github.com/Azure/SQL-Connectivity-Checker/
 or use a machine with Internet access to see how to run this from machines without Internet. See how at https://github.com/Azure/SQL-Connectivity-Checker/
 ```
It seems like the script URL for the Advanced connectivity policy tests script defined with HTTP instead of HTTPS protocol. https://github.com/Azure/SQL-Connectivity-Checker/blob/fd34a078493d16d768d22d09bccc1473858be379/AzureSQLConnectivityChecker.ps1#L1040

**Comment by ThilinaManamgoda** [comment]
Also in the configuration snippet given in the README instruction, the following line should should changed to
`$scriptUrlBase = 'https://raw.githubusercontent.com/Azure/SQL-Connectivity-Checker/master'` 
from 
`$scriptUrlBase = 'http://raw.githubusercontent.com/Azure/SQL-Connectivity-Checker/master`

**Comment by vitomaz-msft** [comment]
Apologies for the delay getting back to you.
Thank you for the feedback, fixed on v1.35
